libcontainer/cgroups/systemd/v1.go

```diff
@@ -485,3 +485,11 @@ func (m *legacyManager) GetFreezerState() (configs.FreezerState, error) {
 	}
 	return freezer.(*fs.FreezerGroup).GetState(path)
 }
+
+func (m *legacyManager) Exists() bool {
+	path, err := getSubsystemPath(m.cgroups, "devices")
```

Comment:
`getSubsystemPath` is super expensive, it parses the whole `/proc/self/mountinfo`.

I think you should be using `m.Path("devices")` here.

Reply:
done